package.json

```diff
@@ -39,9 +39,14 @@
   "keywords": [
     "atom",
     "plugin",
-    "typescript",
+    "issues",
     "github",
-    "gitlab"
+    "gitlab",
+    "projects",
+    "productivity",
+    "tool",
+    "developer",
+    "internet"
```

Comment:
lol I don't think we need this here

(this is how PR reviews will work, I can leave notes on individual lines and request changes, then when you push a fix to it the comment on the line should collapse and you can re-request a review, try it out with this as the example.)